src/app/maker/maker-history-shifts/maker-history-shifts.component.ts

```diff
@@ -9,9 +9,9 @@ import { ShiftService } from 'src/app/shift.service';
   styleUrls: ['./maker-history-shifts.component.scss'],
```

Comment:
What's the story behind the changes to this file? Honestly just out of curiosity

Reply:
stripping down extra layers.  scrolling functionality still exists